cypress/integration/SiteWideSearchAppIntegration.feature

```diff
@@ -0,0 +1,156 @@
+        Given user is navigating to "/search/results?swKeyword=pact"
+        And screen breakpoint is set to "mobile"
+        Then the system returns the results page for "pact"
+        And the "H1" title is "NCI Search Results"
+        And the "H3" title is "Results for: pact"
+        And the best bet box is displayed
+        And the "H2" title is "Best Bets for Partnership for Accelerating Cancer Therapies (PACT)"
+        And the best bets title is a link
+        And the "H4" title "Results 1-20 of XXXXX for: pact" is displayed
+        And both pagers are displayed
+        And the results are displayed with each title containing a link
+
+    Scenario: Negative test:  No search results page
+        Given user is navigating to "/search/results?swKeyword=waowao"
+        And screen breakpoint is set to "desktop"
+        And the "H1" title is "NCI Search Results"
+        And the "H3" title is "0 results found for: waowao"
+        Then the "H3" text is "Please check your spelling or try another search using a different word."
+
+    Scenario Outline: Edge case: search results with 2/3 and bin on English Desktop and Spanish mobile sites.
+        Given user is navigating to "<url>"
+        And screen breakpoint is set to "<breakpoint>"
+        And the "<titleTag>" title is "<title>"
+        And both pagers are displayed
+        And the results are displayed with each title containing a link
+
+        Examples:
+            | breakpoint | url                                      | titleTag | title                |
+            | desktop    | /search/results?swKeyword=bin            | H3       | Results for: bin     |
+            | mobile     | /espanol/buscar/resultados?swKeyword=2/3 | H3       | Resultados para: 2/3 |
+
+    Scenario: Edge case: Fetching more than 30000 results on English site.
+        Given user is navigating to "/search/results/?swKeyword=cancer&page=1518&pageunit=20"
+        And screen breakpoint is set to "desktop"
+        Then the system returns the results page for "cancer"
+        And the "H1" title is "NCI Search Results"
+        And the "H3" title is "Results for: cancer"
+        And the "H4" title "Results XXXXX-XXXXX of XXXXX for: cancer" is displayed
+        And both pagers are displayed
+        And the results are displayed with each title containing a link
+
+    Scenario: Edge case:Display less than 20 results does not show the pager.
+        Given user is navigating to "/search/results?swKeyword=ert"
+        And screen breakpoint is set to "desktop"
+        Then the system returns the results page for "ert"
+        And the "H1" title is "NCI Search Results"
+        And the "H3" title is "Results for: ert"
+        And the dictionary definition box is displayed
+        And the "H2" title is "Definition:"
+        Then the "Show full definition" button within the dictionary definition is displayed
+        And the "H4" title "Results 1-10 of XXXXX for: ert" is displayed
```

Comment:
Why title check step is different from the generic `And the "tag" title is "something:"`?